### ISSUE TITLE
 find() timed out

### ISSUE THREAD

**Issue by davidsarkany** [codetheweb/tuyapi#557]
**Describe the bug**
I got `(node:3772) UnhandledPromiseRejectionWarning: Error: find() timed out. Is the device powered on and the ID or IP correct?` error with the sample code.

**To Reproduce**
I tried to follow the setup guide, but I can't find the "IoT Data Analytics"-on the list.
![image](https://user-images.githubusercontent.com/9111944/147709585-5914a061-a9f5-45c8-a3c8-8190d17c4dde.png)
I skipped this API and followed the steps.
I got device keys with the `tuya-cli wizard` .
```
[
  { name: 'TV', id: 'bf145fe09304xxxxxxxxxxx', key: 'f3dab881yyyyyyyyy' }
]
```
I put id and key the sample code, then I got an error.

**Debug Output**
```
D:\DEV\tv>set DEBUG=* & node index.js
  TuyAPI Finding missing IP undefined or ID bf145fe09304xxxxxxxxxxx +0ms
(node:14096) UnhandledPromiseRejectionWarning: Error: find() timed out. Is the device powered on and the ID or IP correct?
    at D:\DEV\tv\node_modules\tuyapi\index.js:824:13
    at Timeout._onTimeout (D:\DEV\tv\node_modules\p-timeout\index.js:25:13)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:14096) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:14096) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Desktop (please complete the following information):**
 - OS: **Windows**
 - OS Version **10**
 - Node Version **v14.17.0**

**Additional context**
I tried to send command with the online debugger (https://eu.iot.tuya.com/), and it's works. (My LED turned on.)

**Comment by codetheweb** [comment]
> I tried to follow the setup guide, but I can't find the "IoT Data Analytics"-on the list.

It sounds like this has been removed, I updated the setup instructions.

Try manually finding your device's IP and passing it in to the constructor instead of using `.find()` (`.find()` may not work with all network setups).

> I tried to send command with the online debugger (eu.iot.tuya.com), and it's works. (My LED turned on.)

There might be some confusion. This library is for local network control of devices while the Tuya web interface controls them through Tuya's cloud services.
